src/targets/services/helpers/model.js

```diff
@@ -105,6 +82,26 @@ export class Model {
     }
   }
 
+  train(docs) {
```

Comment:
It's ok to keep it simple for now, but eventually, we should use the actual banks categorization: https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/src/libs/categorization/localModel/classifier.js#L148

Reply:
More than simplifying, it allows secret sharing. `classificator` used by banks uses [a sparse matrix and an evolutive vocabulary](https://github.com/Wozacosta/classificator/blob/master/lib/classificator.js#L201), which are incompatible with secret sharing